plugins/woocommerce/includes/class-wc-product-simple.php

```diff
@@ -74,4 +74,27 @@ public function add_to_cart_description() {
 
 		return apply_filters( 'woocommerce_product_add_to_cart_description', sprintf( $text, $this->get_name() ), $this );
 	}
+
+	/**
+	 * Get the add to cart button success message - used to update the mini cart live region.
+	 *
+	 * @return string
+	 */
+	public function add_to_cart_success_message() {
+		/* translators: %s: Product title */
+		$text = $this->is_purchasable() && $this->is_in_stock() ? __( '&ldquo;%s&rdquo; has been added to your cart', 'woocommerce' ) : '';
+
+		if ( ! empty( $text ) ) {
+			$text = sprintf( $text, $this->get_name() );
+		}
+
+		/**
+		 * Filter product add to cart success message.
+		 *
+		 * @since 8.9.2
```

Comment:
This is a newly introduced filter, correct?

```suggestion
		 * @since 9.2.0
```